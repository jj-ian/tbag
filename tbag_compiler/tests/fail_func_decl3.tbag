true {
        testvar();
        endgame;
}

func int testvar(int j) {
        int [10] i;
        return "hello";
}

