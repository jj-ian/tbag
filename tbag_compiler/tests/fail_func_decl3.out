Fatal error: exception Failure("Return type of expression does not match return
type of function")
